### DISCUSSION TITLE
hide navigation bar

### DISCUSSION THREAD

**Discussion by benmaxyoung** [beeware/toga#2321] [Q&A]
Is it possible to support hiding the navigation bar on mobile?

**Comment by freakboy3742** [comment]
At present, there's no API to do this; but PR #2244 is currently working on an API that would allow this. #1870 is the ticket tracking the specific feature request.

**Reply by benmaxyoung** [reply]
OK I also think `title=None` is the best way to hide the navigation bar. Other styles can use custom navigation bars.

**Comment by benmaxyoung** [comment]
Is there support for swipe/long press gestures now?

**Reply by freakboy3742** [reply]
Not at present.
